src/backend/base/langflow/components/data/Gmail.py

```diff
@@ -0,0 +1,196 @@
+import base64
+import re
+import json
+from typing import Any, Iterator, List
+from google.oauth2.credentials import Credentials
+from googleapiclient.discovery import build
+from langflow.custom import Component
+from langflow.inputs import MessageTextInput
+from langflow.io import SecretStrInput
+from langflow.template import Output
+from langflow.schema import Data
+from langchain_community.chat_loaders.gmail import GMailLoader
```

Comment:
It seems this is deprecated. Could you take a look at https://github.com/langchain-ai/langchain-google/tree/main/libs/community/langchain_google_community

Reply:
Sure; I did nottice it. 
Updating it to use Gmail Loaders from langchain_google_community